cpython/Makefile

```diff
@@ -98,21 +107,30 @@ $(SQLITEBUILD)/libsqlite3.la: $(SQLITETARBALL)
 	)
 
 
-$(BUILD)/Makefile: $(BUILD)/.patched $(ZLIBBUILD)/.patched $(SQLITEBUILD)/libsqlite3.la
+$(BZIP2BUILD)/libbz2.a: $(BZIP2TARBALL)
+	[ -d $(ROOT)/build ] || (mkdir $(ROOT)/build)
+	tar -C $(ROOT)/build/ -xf $(BZIP2TARBALL)
+	( \
+		cd $(BZIP2BUILD); \
+		emmake make CC=emcc CFLAGS="-Wall -Winline -O2 -fomit-frame-pointer -D_FILE_OFFSET_BITS=64" AR=emar RANLIB=emranlib libbz2.a; \
```

Comment:
I would rather we used default flags here (or at least those specified by some shared ENV variable). Are all of these necessary for the build?

Reply:
OK, I'm in the process of rebuilding my local environment so I can't test, but I went ahead and pushed 78d514e and we can see if the CI tests pass.

Reply:
Tests failed.

```pytb
TypeError: Cannot read property 'apply' of undefined
    at _BZ2_bzDecompressInit (http://127.0.0.1:33409/pyodide.asm.js:8:44947)
```

Looks like at least one of those options is needed.

Reply:
Ok I removed the CFLAGS. The tool definitions seem to be necessary. 

Ready to merge?